tools/build_script_generator/cmake/resources/main.cpp.in

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
remove this file, this was a development thing.

Reply:
Done.